src/ui/index.jsx

```diff
@@ -101,6 +104,7 @@ Lbryio.setOverride(
         // @endif
         // @if TARGET='web'
         const { auth_token: authToken } = cookie.parse(document.cookie);
+        console.log('retrieved auth', authToken);
```

Comment:
Impressive inclusion for such a short PR :grinning: 

Any reason linter did not catch?

Reply:
🤦‍♂ 

I turned that lint rule off because it was a pain with all the console logs in our electron files. I'll add it back.